artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelManager.java

```diff
@@ -34,7 +34,7 @@
 
    private static final Logger logger = Logger.getLogger(JChannelManager.class);
 
-   private Map<String, JChannelWrapper> channels;
```

Comment:
why the static change here? this changes the semantic completely from what you needed.